redis/google/cloud/redis_v1/proto/cloud_redis.proto

```diff
@@ -18,6 +18,7 @@ syntax = "proto3";
 package google.cloud.redis.v1;
 
 import "google/api/annotations.proto";
+import "google/api/resource.proto";
```

Comment:
This proto was added 5 days ago, not sure if that's related to the imports failing: https://github.com/googleapis/googleapis/blob/master/google/api/resource.proto

Reply:
googleapis-common-protos 1.5.8 does not have resource_pb2. Following up to see how to get that updated.

Reply:
I think this is usually removed during sanitization?